### ISSUE TITLE
And where is explanation what channel is? 

### ISSUE THREAD

**Issue by dmitrypotenko** [bloomreach/docs-feedback#26]
https://documentation.bloomreach.com/14/library/concepts/hst-configuration-model/sites--channels-configuration.html

**Comment by nvankampenhout** [comment]
Hi, we lack a single page explaining the domain model in brief and we will add this to our TODO list of documentation improvements.

In the meantime, let me point you to the following pages that touch on what a channel is:

https://documentation.bloomreach.com/14/library/concepts/hst-configuration-model/introduction.html
https://documentation.bloomreach.com/14/library/concepts/channels/channel-manager.html

In short, a channel is a a site that can be configured and edited in the Experience manager UI.

Hope this helps. Thanks for your feedback!
